src/main/java/com/zachary_moore/objects/WrappedObjectHelper.java

```diff
@@ -29,4 +32,25 @@ static WrappedObject getWrappedObject(String originatingKey,
         }
         return wrappedObject;
     }
+
+    public static String getPath(WrappedObject element) {
```

Comment:
can we make this a list instead of string separated by a '.' as discussed offline